snowflake/assets/dashboards/organization_metrics.json

```
@@ -0,0 +1,982 @@
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "avg:snowflake.organization.balance.on_demand_consumption{$billing_account,$scope} by {contract_number,currency}"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "dog_classic"
+                                    }
+                                }
+                            ],
+                            "show_legend": true,
+                            "title": "Consumption at on Demand Prices",
+                            "title_align": "left",
+                            "title_size": "16",
+                            "type": "timeseries"
+                        },
+                        "id": 5834465578288372,
+                        "layout": {
+                            "height": 2,
+                            "width": 6,
+                            "x": 6,
+                            "y": 2
+                        }
+                    }
+                ]
+            },
+            "id": 3702666235793118,
+            "layout": {
+                "height": 5,
+                "width": 12,
+                "x": 0,
+                "y": 21
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Usage in Currency",
+                "type": "group",
+                "widgets": [
+                    {
+                        "definition": {
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "aggregator": "avg",
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "avg:snowflake.organization.currency.usage{$billing_account,$scope} by {usage_type}"
```

Should we add the `currency` tag to the aggregation scope here? Since the widget group is titled "Usage in Currency"

```suggestion
                                            "query": "avg:snowflake.organization.currency.usage{$billing_account,$scope} by {currency, usage_type}"
```

---

Yes I can do that!

---

Updated!